doc/R-package/index.rst

```diff
@@ -13,7 +13,17 @@ You have found the XGBoost R Package!
 Get Started
 ***********
 * Checkout the :doc:`Installation Guide </install>` contains instructions to install xgboost, and :doc:`Tutorials </tutorials/index>` for examples on how to use XGBoost for various tasks.
-* Read the `API documentation <https://cran.r-project.org/web/packages/xgboost/xgboost.pdf>`_.
+* Read the `API documentation <../r_docs/R-package/docs/index.html>`__
```

Comment:
Perhaps it could mention that these are the docs for the latest github version, while the ones at CRAN are for the latest release. Or will it have a version switcher like the sphinx docs?

Reply:
Done.